react-app/src/features/profile/me/index.tsx

```diff
@@ -186,6 +192,31 @@ export const MyProfile = () => {
       window.scrollTo(0, 0);
     }
   };
+
+  const handleRoleStatusClick = (status: string, territory: StateCode) => {
+    if (status === "pending" && isNewUserRoleDisplay) return setSelfWithdrawPending(true);
+    return setSelfRevokeState(territory);
+  };
+
+  const { dialogTitle, dialogBody, ariaLabelledBy } = getConfirmationModalText(
+    selfRevokeState,
+    selfWithdrawPending,
+  );
+
+  const handleDialogOnAccept = async () => {
+    if (selfRevokeState) await handleSelfRevokeAccess();
+    else {
+      // TODO: add in the logic to remove pending request move state change into that function
```

Comment:
I see the 6.1AC has been struck out, is that because the functionality to process the acceptance of the cancel role is moving to a new story?